win32ss/pch.h

```diff
@@ -13,6 +13,10 @@
 #define _NO_COM
 #define STRICT
 
+#ifdef __cplusplus
+    #define NONAMELESSUNION /* C++ forbids nameless struct/union */
+#endif
```

Comment:
Somehow we are able to cope with that in other C++ modules that use SDK headers, so there should be a more elegant solution.

Reply:
SDK headers are using `DUMMYSTRUCTNAME` as this PR does.

Reply:
- `__MINGW_EXTENSION`
- `_MSC_EXTENSIONS`

Reply:
I used `_ANONYMOUS_STRUCT` macro and expanded the nameless members.